singlefactorauth/build.gradle

```diff
@@ -37,7 +37,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.6.1'
 
     implementation 'org.web3j:crypto:4.8.9-android'
-    implementation 'org.torusresearch:torus-utils-java:3.4.0'
+    implementation 'com.github.grvgoel81:torus-utils-java:7.0.0'
```

Comment:
why this repo and not web3auth's?

Reply:
Its not final since torus-utils-java is under review.

Reply:
We should get that merged in first and then update it in this PR.